modules/mobile/app/workers/mobile/v0/weekly_maintenance_window_logger.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'date'
+
+module Mobile
+  module V0
+    class WeeklyMaintenanceWindowLogger
+      include Sidekiq::Worker
+
+      sidekiq_options retry: 3, unique_for: 1.week
+
+      def perform
+        upstream_maintenance_windows = ::MaintenanceWindow.end_after(Time.zone.now)
```

Comment:
I would suggest we do this a bit differently. Maybe something like:
`::MaintenanceWindow.where('created_at >= ?', DateTime.now - 7.days)`

The way it's currently written, if a maintenance window is created after we log and ends before we log again, it will not be picked up. The job that creates maintenance windows (app/workers/pager_duty/poll_maintenance_windows.rb) runs every three minutes, so I'm not sure we can assume that these are created very far in advance.